### PR TITLE
Clarify Instruction to state that characters that are not letters (in…

### PR DESCRIPTION
**Pull request by emiliolg**
…cluding digits, etc) should be removed.

Add a test that showed that more clearly.

# pull request

Doing Mentoring I noticed that instructions are not clear about removing digits and so on. 
I checked the C# track and instructions were clear. So I update the Java instructions to reflect that.
I also add a test that an additional test to show removing no letters while keeping them.

Updae hints.md

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/concept/squeaky-clean/src/test/java/SqueakyCleanTest.java [hunk 1]
```diff
@@ -38,6 +38,10 @@ public void ctrl() {
     public void string_with_no_letters() {
         assertThat(SqueakyClean.clean("\uD83D\uDE00\uD83D\uDE00\uD83D\uDE00")).isEmpty();
     }
+    @Test
+    public void remove_no_letters() {
```
Comment (andrerfcsantos) [inline exercises/concept/squeaky-clean/src/test/java/SqueakyCleanTest.java:42]:
```suggestion

    @Test
    public void keep_only_letters() {
```

Maybe this is a more appropriate name for the test?

**Review by andrerfcsantos — CHANGES_REQUESTED** [review]
Looks cool, left some minor suggestions.

Note that when changing the tasks, [hints.md](https://github.com/exercism/java/blob/main/exercises/concept/squeaky-clean/.docs/hints.md) should also be modified, and the section names must match the names for the tasks in the instructions.

**Review by emiliolg — COMMENTED** [review]
Fix hints.md

**Review by andrerfcsantos — APPROVED** [review]
Thanks for this.

**Pull request merged**